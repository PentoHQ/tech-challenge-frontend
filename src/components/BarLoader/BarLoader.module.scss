@import 'common.scss';
@import 'colors.scss';

.Container {
  display: flex;
  height: 100%;
  justify-content: center;
  align-items: center;
  font-family: $primaryFont;
  font-weight: $mediumFontWeight;
  font: $text14Regular;
}

.Icon {
  width: 50px;
  height: 50px;
  color: $primary90;
}
