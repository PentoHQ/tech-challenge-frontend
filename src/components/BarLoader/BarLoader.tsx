import { BarLoaderIcon } from 'assets'
import React from 'react'
import styles from './BarLoader.module.scss'

export const BarLoader: React.FC = () => {
  return (
    <div className={styles.Container}>
      <BarLoaderIcon className={styles.Icon} />
    </div>
  )
}
