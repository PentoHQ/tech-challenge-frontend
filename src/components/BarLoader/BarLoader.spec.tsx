import React from 'react'
import { render } from '@testing-library/react'

import { BarLoader } from './BarLoader'

describe('<Loader/>', () => {
  it('renders', () => {
    const { container } = render(<BarLoader />)

    expect(container).toMatchSnapshot()
  })
})
