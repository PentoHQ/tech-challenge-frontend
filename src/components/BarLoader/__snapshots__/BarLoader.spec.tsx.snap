// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Loader/> renders 1`] = `
<div>
  <div
    class="Container"
  >
    <svg
      class="Icon"
    >
      bars.svg
    </svg>
  </div>
</div>
`;
